qiskit_machine_learning/datasets/dataset_helper.py

```diff
@@ -19,16 +19,27 @@
 from sklearn import preprocessing
 
 
-def discretize_and_truncate(data, bounds, num_qubits, return_data_grid_elements=False,
+def discretize_and_truncate(data, min_max_bin_centers, num_qubits, return_data_grid_elements=False,
                             return_prob=False, prob_non_zero=True):
     """
     Discretize & truncate classical data to enable digital encoding in qubit registers
-    whereby the data grid is [[grid elements dim 0],..., [grid elements dim k]]
+    whereby the data grid is [[grid elements dim 0], ..., [grid elements dim k]].
+
+    For each dimension k, the domain is split into (2 ** num_qubits[k]) bins equally spaced and
+    equally sized, each centered in min_max_bin_centers[k, 0], ..., min_max_bin_centers[k, 1]. Bins
+    have size equal to
+    (min_max_bin_centers[k, 1] - min_max_bin_centers[k, 0])/(2 ** num_qubits[k] - 1).
+    Notice that:
+    * Every sample in data that falls out of the bins is discarded.
+    * The leftmost bin extends both to the left and to the right around its center,
+    therefore min_max_bin_centers[k, 0] is not the left bound for truncation, but only the center of
+    the leftmost bin. Similar considerations hold for min_max_bin_centers[k, 1] on the right.
 
     Args:
         data (list or array or np.array): training data (int or float) of dimension k
-        bounds (list or array or np.array):  k min/max data values
-            [[min_0,max_0],...,[min_k-1,max_k-1]] if univariate data: [min_0,max_0]
+        min_max_bin_centers (list or array or np.array):  k min/max data values
+            [[min_center_0, max_center_0],...,[min_center_k-1, max_center_k-1]].
+            If univariate data: [min_center_0, max_center_0]
         num_qubits (list or array or np.array): k numbers of qubits to determine
             representation resolution, i.e. n qubits enable the representation of 2**n
             values [num_qubits_0,..., num_qubits_k-1]
```

Comment:
Could you please reformat the whole docstring to highlight code/math as I suggested?